src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/caseworker/MoreTimeRequestedCallbackHandler.java

```diff
@@ -0,0 +1,269 @@
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.RESPONSE_MORE_TIME;
+import static uk.gov.hmcts.cmc.claimstore.services.ccd.Role.CASEWORKER;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationReferenceBuilder.MoreTimeRequested.referenceForClaimant;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationReferenceBuilder.MoreTimeRequested.referenceForDefendant;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.CLAIMANT_NAME;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.CLAIMANT_TYPE;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.CLAIM_REFERENCE_NUMBER;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.DEFENDANT_NAME;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.FRONTEND_BASE_URL;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.RESPONSE_DEADLINE;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDate;
+
+@Service
+@ConditionalOnProperty("feature_toggles.ctsc_enabled")
+public class MoreTimeRequestedCallbackHandler extends CallbackHandler {
+
+    private static final List<Role> ROLES = Collections.singletonList(CASEWORKER);
+
+    private static final List<CaseEvent> EVENTS = Collections.singletonList(RESPONSE_MORE_TIME);
+
+    private final EventProducer eventProducer;
+    private final ResponseDeadlineCalculator responseDeadlineCalculator;
+    private final MoreTimeRequestRule moreTimeRequestRule;
+    private final CaseDetailsConverter caseDetailsConverter;
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+    private final GeneralLetterService generalLetterService;
+    private final UserService userService;
+    private final String generalLetterTemplateId;
+
+    private static final String PREVIEW_SENTENCE = "The response deadline will be %s .";
+    private static final String ERROR_MESSAGE =
+        "There was a technical problem. Nothing has been sent. You need to try again.";
+    private static final String STANDARD_DEADLINE_TEXT = "You’ve been given an extra 14 days to respond to the"
+            + " claim made against you by %s.\n"
+            + "\n"
+            + "You now need to respond to the claim before 4pm on %s.\n"
+            + "\n"
+            + "If you don’t respond, you could get a County Court Judgment (CCJ). This may make it harder to get "
+            + "credit, such as a mobile phone contract, credit card or mortgage.";
```

Comment:
As java is platform independent it would be better to get the new line character from the system i.e. ```String newLine = System.getProperty("line.separator");```

Also it is safer to use the String.join method here so that any future devs don't accidentally remove the new line char, which is easily done. i.e 
```suggestion
    private static final String STANDARD_DEADLINE_TEXT = String.join(newLine,
    "You’ve been given an extra 14 days to respond to the claim made against you by %s.",
            "",
            "You now need to respond to the claim before 4pm on %s.",
            "",
            "If you don’t respond, you could get a County Court Judgment (CCJ). This may make it harder to get "
            + "credit, such as a mobile phone contract, credit card or mortgage.");
```

Reply:
Since you're using {{format}} you can also use {{%n}} instead of {{\n}}, which will use the system's configured new line char(s).